enterprise/utils.py

```diff
@@ -601,15 +601,29 @@ def wrapper(request, *args, **kwargs):
 
         enterprise_uuid = kwargs['enterprise_uuid']
         enterprise_customer = get_enterprise_customer_or_404(enterprise_uuid)
+        provider_id = enterprise_customer.identity_provider or ''
+        sso_provider = None
+
+        try:
+            sso_provider = get_identity_provider(provider_id)
+        except ValueError:
+            pass
+
+        decorator_already_processed = request.GET.get('session_cleared') == 'yes'
```

Comment:
Do you need to check the value of param here or is the fact that it exists enough?